bower.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "express-couchuser-fluid-frontend",
+  "version": "0.0.0",
+  "homepage": "https://github.com/the-t-in-rtf/express-couchuser-fluid-frontend",
+  "authors": [
+    "Tony <tony@raisingthefloor.org>"
+  ],
+  "description": "A series of fluid components that provide a frontend to the express-couchuser API.",
+  "moduleType": [
+    "amd"
+  ],
+  "license": "See LICENSE file for details.",
+  "private": true,
+  "main": [
+    "./src/js/forgot.js",
+    "./src/js/login.js",
+    "./src/js/reset.js",
+    "./src/js/signup.js",
+    "./src/js/user-controls.js",
+    "./src/js/verify.js"
+  ],
+  "ignore": [
+    "**/.*",
+    "Gruntfile.js",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "gpii-handlebars": "git://github.com/the-t-in-rtf/gpii-handlebars.git",
+    "foundation":              "zurb/bower-foundation#~5.3.1"
```

Comment:
Turns out my IDE had different indentation settings for JSON content.  I changed them to match the Javascript conventions and reformatted.